hands-on/partials/_launch-app.html.md

```diff
@@ -1,15 +1,23 @@
 Fly.io allows you to deploy any kind of app as long as it is packaged in a **Docker image**. That also means you can just deploy a Docker image and as it happens we have one ready to go in `flyio/hellofly:latest`. 
 
-Each Fly.io application needs a `fly.toml` file to tell the system how we'd like to deploy it. That file can be automatically generated with the `flyctl launch` command. 
+Each Fly.io application needs a `fly.toml` file to tell the system how we'd like to deploy it. That file can be automatically generated with the `flyctl launch` command, which will ask a few questions to set everything up. Let's run through it now: 
 
 ```cmd
 flyctl launch --image flyio/hellofly:latest
 ```
 ```output 
+? App Name (leave blank to use an auto-generated name):
+```
+Here you can enter the name of the app.
```

Comment:
Maybe you can also add in here some details on the app name value validation, I think app names are limited only to: lowercase letters, dashes, and numbers
![image](https://user-images.githubusercontent.com/33826441/192500714-f4fbe454-b714-44ae-a03c-5a616f4f80ba.png)

Reply:
Good idea, has been added